packages/botonic-plugin-text-classification/README.md

```diff
@@ -0,0 +1,36 @@
+# Botonic Plugin Text Classification
+
+## What does this plugin do?
+
+Botonic Plugin Text Classification uses the trained models defined by the user to classify the input text.
+
+After detecting the text locale, and using the corresponding model to predict the class, extra information about the user's intent is added to the input message.
+
+## Install the plugin
+
+From your project, install the plugin by using the following command:
+
+```bash
+npm install @botonic/plugin-text-classification
+```
+
+> **Note**: Windows users should first use the command
+> `npm install --global --production windows-build-tools --vs2015`
+
+## Require the plugin
```

Comment:
Requiring the plugin (for the docs I've seen so far, it's common to use this tense)